src/Country/UserAssigned/TravelDocumentAlpha2.php

```diff
@@ -23,4 +23,7 @@ enum TravelDocumentAlpha2: string
 
     /** Used by ICAO when digitally signing a master list */
     case International_Civil_Aviation_Organization = 'IA';
+
+    /** Used by European Commission and many institutions in EU countries */
+    case Kosovo = 'XK';
```

Comment:
This specification is for country codes used in travel documents (See comment at the top of this enum, referencing https://www.icao.int/publications/Documents/9303_p3_cons_en.pdf, page 29). This "XK", seeing your comment is a Top Level Domain, not related to Travel documents. It is also not referenced in the root zone database https://www.iana.org/domains/root/db, but is in the reserved range for TLDs. It is also a user assigned code outside the specification for country codes, so not referenced in the CountryCodeAlpha2 specification.

Are you handling travel documents in any way? If so, have you seen a MRTD with this code? Looking at the source document for this specification, The Republic of Kosovo actually does issue its own documents, and the code for that should be KS.